packages/web-client/src/components/Navbar/Categories.tsx

```diff
@@ -55,7 +55,7 @@ const Categories = () => {
           {...attributes.popper}
         >
           {categories.map((category) => (
-            <Link href="/categories/[slug]" as="/categories/slug">
+            <Link href={`/categories/${category.slug}`} as={`/categories/${category.slug}`}>
```

Comment:
href should be a string, same as the folder structure
`href="/categories/[slug]"`